src/sentry/seer/anomaly_detection/delete_rule.py

```diff
@@ -0,0 +1,86 @@
+import logging
+from typing import TYPE_CHECKING, cast
+
+from django.conf import settings
+from urllib3.exceptions import MaxRetryError, TimeoutError
+
+from sentry.conf.server import SEER_ALERT_DELETION_URL
+from sentry.models.organization import Organization
+from sentry.net.http import connection_from_url
+from sentry.seer.anomaly_detection.types import DeleteRuleRequest
+from sentry.seer.signed_seer_api import make_signed_seer_api_request
+from sentry.utils import json
+from sentry.utils.json import JSONDecodeError
+
+logger = logging.getLogger(__name__)
+
+seer_anomaly_detection_connection_pool = connection_from_url(
+    settings.SEER_ANOMALY_DETECTION_URL, timeout=settings.SEER_DEFAULT_TIMEOUT
+)
+
+if TYPE_CHECKING:
+    from sentry.incidents.models.alert_rule import AlertRule
+
+
+def delete_rule_in_seer(alert_rule: "AlertRule") -> bool:
```

Comment:
Why can't we just import it normally?

Reply:
Circular imports (this function is used in alertrule.py, from which we import the AlertRule model)

Reply:
ya - FWIW we should _probably_ be doing this any time we're using an import only for types? 🤷